go.mod

```diff
@@ -69,3 +69,5 @@ require (
 	k8s.io/api v0.21.3
 	k8s.io/apimachinery v0.21.3
 )
+
+replace github.com/containers/common => github.com/saschagrunert/common v0.40.2-0.20210730110303-fc86f687be82
```

Comment:
TODO: remove and vendor latest c/common master if https://github.com/containers/common/pull/700 got merged